docs/source/reference.adoc

```diff
@@ -360,3 +360,109 @@ because generating a new token is an expensive operation. Get the access token i
 curl -H "Content-Type: application/json" -X POST -d '{"refresh_token":"<REFRESH_TOKEN>"}' https://auth.openshift.io/api/token/refresh
 
 As a result, you will get a JSON with an access token which you can use in your test.
+
+== Token validation
+
+=== The JWT contents
+
+The token is likely to contain the following claims.
```

Comment:
`likely` or `has to` ? It may contain more claims, but AFAIK, these ones are required, aren't they?

Reply:
Well... The token in this example is an Access token. All of these claims will be there.
But we also generate and use Refresh Tokens and Service Account Tokens. They will have different set of claims.
But all of them will have "kid" header and "exp", "sub", "jti" claims.
We will also have PRT tokens but they are Access Tokens + extra AuthZ claims.

Reply:
Yes, I plan to expand this portion in future to show what different kinds of tokens ( service account, user token, refresh token, RPTs) would contain.